docs/using-radicle/identity.md

```diff
@@ -0,0 +1,65 @@
+---
+id: identity
+title: Manage your Radicle identity
+---
+
+Your Radicle identity is your ticket to hosting and collaborating on code in the Radicle network, but it's not a fixed
+entity. You can manage multiple identities, creating/delete them at will, and even edit them to a degree.
+
+Many of the actions below require your personal URN, which you can find using `rad self`.
+
+## View your active identity
+
+Run `rad self` to see details on how you're identified on the Radicle network, including your:
+
+- Active [display name](#edit-your-display-name)
+- Personal URN
+- Peer ID
+- Hashed and full encryption keys
+- File locations of your Git monorepo and encryption keys
+
+## Create a new identity
+
+Add new identities at any time. For example, use different identities for the personal projects you host on Radicle
+versus your professional work for a company or DAO.
+
+```
+rad auth --init
+```
+
+## Switch between identities
+
+If you have multiple identities, switch between them with `rad auth`.
+
+Choose the identity you want active, enter your passphrase, and hit **Enter**.
+
+## Edit your display name
+
+Change your _display name_, which is how you're identified on the Radicle web app (not the same as your Peer ID or
+personal URN!):
+
+```
+rad edit
+```
+
+Change the `"name"` value to your new display name in the file that appears.
+
+```
+{
+  "https://radicle.xyz/link/identities/person/v1": {
+    "name": "your-display-name"
+  }
+}
+```
+
+Re-run `rad self` to see that your **Name** changed, but your personal URN and Peer ID stayed the same.
+
+## Delete your identity
+
+Delete any identity on your local machine with the following:
+
+```
+rad rm YOUR-PERSONAL-URN
```

Comment:
Would be great to also mention the `no-passphrase` option in case the user lost their password.

Reply:
Is that option available on v0.6.1? I'm getting an `invalid option` warning.

I'm thinking that, because the current version (via Homebrew or apt packages) is to install v0.6.1, we should probably stick with what's available on that version for now.

Reply:
As a temporary workaround, I did add a section about deleting the folders manually.